docs-md/tutorials/embedded-connect.md

```diff
@@ -0,0 +1,334 @@
+
+We can easily achieve this result using ksqlDB by simply joining the ``driverLocations`` stream with the ``drivers`` table stored in PostgreSQL:
+
+```sql
+CREATE STREAM enrichedDriverLocations AS
+  SELECT
+    dl.driver_id       AS driver_id,
+    dl.latitude        AS latitude,
+    dl.longitude       AS longitude,
+    dl.speed           AS speed,
+    jdbc.make          AS make,
+    jdbc.model         AS model,
+    jdbc.year          AS year,
+    jdbc.license_plate AS license_plate,
+    jdbc.rating AS rating
+  FROM driverLocations dl JOIN drivers jdbc
+    ON dl.driver_id = jdbc.driver_id
+  EMIT CHANGES; 
+```
+
+13. Create rendezvous stream
+----------------------------
+
+Putting all of this together, we will now create a final stream that the ride sharing app can use to facilitate a driver-rider rendezvous in real time. This stream is defined by a query that joins together rider and driver location updates, resulting in a contextualized output that the app may use to show the rider their driver’s position as the rider waits to be picked up.
+
+Our rendezvous stream also includes human-friendly information describing the driver’s vehicle for the rider, and even computes (albeit naively) the driver’s estimated time of arrival (ETA) at the rider’s location:
+
+```sql
+CREATE STREAM rendezvous AS
+  SELECT
+    e.license_plate AS license_plate,
+    e.make          AS make,
+    e.model         AS model,
+    e.year          AS year,
+    e.latitude      AS vehicle_lat,
+    e.longitude     AS vehicle_long,
+    GEO_DISTANCE(e.latitude, e.longitude, r.latitude, r.longitude) / e.speed AS eta
+  FROM enrichedDriverLocations e JOIN riderLocations r WITHIN 1 MINUTE
+    ON e.driver_id = r.driver_id
+  EMIT CHANGES;
+```
+
+14. Start two ksqlDB CLI sessions
+---------------------------------
+
+Run this command to twice to open two separate ksqlDB CLI sessions. Will we use both of these sessions in the steps to follow. Note that if you still have a CLI session open from a previous step, you may reuse that session.
+
+```bash
+docker exec -it ksqldb-cli ksql http://ksqldb-server:8088
+```
+
+15. Run a continuous query
+--------------------------
+
+We’re now going to run a continuous query over the rendezvous stream.
+
+This is the first thing that may feel a bit unfamiliar to you, because the query will never return until it's terminated. It will perpetually push output rows to the client as events are written to the rendezvous stream. Leave the query running in your CLI session for now. It will begin producing output as soon as we write events into ksqlDB:
```

Comment:
```suggestion
This may feel a bit unfamiliar, because the query never returns until you terminate it. The query perpetually pushes output rows to the client as events are written to the rendezvous stream. Leave the query running in your CLI session for now. It will begin producing output as soon as events are written into ksqlDB.
```